src/tbbbind/tbb_bind.cpp

```diff
@@ -85,6 +86,16 @@ class platform_topology {
         // Parse topology
         if ( hwloc_topology_init( &topology ) == 0 ) {
             initialization_state = topology_allocated;
+            #if __HWLOC_RESTRICT_TO_CPUBINDING_TOPOLOGY_FLAG_PRESENT
+            if ( groups_num == 1 &&
+                 hwloc_topology_set_flags(topology,
+                     HWLOC_TOPOLOGY_FLAG_IS_THISSYSTEM |
+                     HWLOC_TOPOLOGY_FLAG_RESTRICT_TO_CPUBINDING
+                 ) != 0
+            ) {
+                return;
+            }
+            #endif
```

Comment:
It seems that we can avoid introducing new macro by having something like:
```suggestion
            if ( groups_num == 1 &&
                 hwloc_topology_set_flags(topology,
                     HWLOC_TOPOLOGY_FLAG_IS_THISSYSTEM 
#if HWLOC_TOPOLOGY_FLAG_RESTRICT_TO_CPUBINDING
                     | HWLOC_TOPOLOGY_FLAG_RESTRICT_TO_CPUBINDING
#endif
                 ) != 0
            ) {
                return;
            }
```
What do you think?

Reply:
What is the purpose of this change?

Reply:
I cannot do it since it is an enum, not a macro.

Reply:
Should not we include it in the macro definition then? Something like:
```cpp
#define __TBBBIND_HWLOC_TOPOLOGY_FLAG_RESTRICT_TO_CPUBINDING_PRESENT \
    (HWLOC_API_VERSION >= 0x20500 && HWLOC_TOPOLOGY_FLAG_RESTRICT_TO_CPUBINDING !=0)
```